### DISCUSSION TITLE
What does "activate subscriber" in the "Create Subscriber" pop-up actually *do*?

### DISCUSSION THREAD

**Discussion by krangor** [buttondown/discussions#31] [Q&A] [answered]
It's not super clear to me whether this:
- automatically makes the email address a subscriber
OR
- sends them an activation email

I notice if I select this I get a "new subscriber" email some time later, and, while I was assuming the former, this makes me wonder if it's doing the latter and sending me the confirm once the subscriber clicks through the activate email.

**Comment by jmduke** [comment, marked as answer]
It's the former, though we really shouldn't send that email if you're the one creating the subscriber — mind sharing the email address so I can take a look as to why that happened (you can email me at justin@buttondown.email if you don't want it to be public!)
